doc/source/whatsnew/v0.17.0.txt

```diff
@@ -334,6 +335,62 @@ Google BigQuery Enhancements
 - The ``generate_bq_schema()`` function is now deprecated and will be removed in a future version (:issue:`11121`)
 - Update the gbq module to support Python 3 (:issue:`11094`).
 
+.. _whatsnew_0170.east_asian_width:
+
+Display Alignemnt with Unicode East Asian Width
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+   Enabling this option will affect to print DataFrame and Series performance (about 2 times slower).
+   Use only when it is actually required.
+
+Some east asian countries use Unicode characters its width is corresponding to 2 alphabets.
+If DataFrame or Series contains these characters, default output cannot be aligned properly.
+
+.. ipython:: python
+
```

Comment:
I would move most of this to the ``options.rst`` and just put a short example here and  a ref to the docs.